src/main/kotlin/no/skatteetaten/aurora/gobo/resolvers/resolvers.kt

```diff
@@ -64,11 +64,11 @@ fun <T> Mono<T>.handleError(sourceSystem: String?) =
 
 private fun WebClientResponseException.readResponse(): String? {
     this.request?.let {
-        logger.error { "Error request url:${it.uri.toASCIIString()}" }
+        logger.debug { "Error request url:${it.uri.toASCIIString()}" }
```

Comment:
Burde denne vært feks info? Litt redd for at vi kan miste litt kilder til feilsøking hvis dette ikke logges til vanlig.

Reply:
Godt poeng. Har endret til info.